### ISSUE TITLE
[kahi_staff_udea_person]

### ISSUE THREAD

**Issue by omazapa** [colav/Kahi_plugins#62]
```
Running plugin: kahi_staff_udea_person
Traceback (most recent call last):
  File "/home/ozapatam/.local/bin/kahi_run", line 37, in <module>
    kahi.run()
  File "/home/ozapatam/.local/lib/python3.11/site-packages/kahi/Kahi.py", line 121, in run
    plugin_instance = plugin_class(config=plugin_config)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ozapatam/.local/lib/python3.11/site-packages/kahi_staff_udea_person/Kahi_staff_udea_person.py", line 37, in __init__
    raise Exception("University of Antioquia not found in database")
Exception: University of Antioquia not found in database
```

**Comment by muzgash** [comment]
To include anything from udea, ror and wikipedia plugins must finish successfully.

**Comment by muzgash** [comment]
Fixed in version 0.0.3a0
